src/GetComments.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+
+export default function GetComments(props) {
+  const { article_id } = props;
+  const [comments, setComments] = useState([]);
+  const [commentsHidden, setCommentsHidden] = useState(true);
+
+  useEffect(() => {
+    fetch(
+      `https://nc-news-itve.onrender.com/api/articles/${article_id}/comments`
+    )
+      .then((response) => {
+        return response.json();
+      })
+      .then((result) => {
+        setComments(result);
+      });
+  }, []);
+
+  function handleClick(event) {
+    event.preventDefault();
+    if (commentsHidden === true) {
+      setCommentsHidden(false);
+    } else {
+      setCommentsHidden(true);
+    }
+  }
+
+  return (
+    <section className="comments">
+      <button className="show-hide-comments" onClick={handleClick}>
```

Comment:
At the moment, if we click this button for an article that has no comments, it just says 'Hide Comments' but none are there. It would be great if we could have something in here to inform the user that there are none available. 👍